sim/paladin/items.go

```diff
@@ -49,36 +53,104 @@ func init() {
 		})
 	})
 
-	
+	// https://www.wowhead.com/classic/item=230272/scrolls-of-blinding-light
+	// Use: Energizes a Paladin with light, increasing melee attack speed by 25%
+	// and spell casting speed by 33% for 20 sec. (2 Min Cooldown)
+	core.NewItemEffect(ScrollsOfBlindingLight, func(agent core.Agent) {
+		character := agent.GetCharacter()
+
+		duration := time.Second * 20
+
+		aura := character.GetOrRegisterAura(core.Aura{
+			Label:    "Blinding Light",
+			ActionID: core.ActionID{SpellID: 467522},
+			Duration: duration,
+			OnGain: func(aura *core.Aura, sim *core.Simulation) {
+				character.MultiplyAttackSpeed(sim, 1.25)
+				character.MultiplyCastSpeed(1.33)
+			},
+			OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+				character.MultiplyAttackSpeed(sim, 1.0/1.25)
+				character.MultiplyCastSpeed(1.0 / 1.33)
+			},
+		})
+
+		spell := character.GetOrRegisterSpell(core.SpellConfig{
+			ActionID: core.ActionID{ItemID: ScrollsOfBlindingLight},
+			Flags:    core.SpellFlagNoOnCastComplete | core.SpellFlagOffensiveEquipment,
+
+			Cast: core.CastConfig{
+				CD: core.Cooldown{
+					Timer:    character.NewTimer(),
+					Duration: time.Minute * 2,
+				},
+				SharedCD: core.Cooldown{
+					Timer:    character.GetOffensiveTrinketCD(),
+					Duration: duration,
+				},
+			},
+
+			ApplyEffects: func(sim *core.Simulation, _ *core.Unit, _ *core.Spell) {
+				aura.Activate(sim)
+			},
+		})
+
+		character.AddMajorCooldown(core.MajorCooldown{
+			Type:     core.CooldownTypeDPS,
+			Priority: core.CooldownPriorityDefault,
+			Spell:    spell,
+		})
+	})
+
+	core.NewItemEffect(HammerOfTheLightbringer, func(agent core.Agent) {
+		character := agent.GetCharacter()
+		blazefuryTriggerAura(character, 465412, core.SpellSchoolHoly, 4)
+		crusadersZealAura465414(character)
+	})
+
+	// https://www.wowhead.com/classic/item=229749/truthbearer
```

Comment:
Looks like the wowhead looks for truthbearer are backwards. The one here is for the 2hander and the one below is for the 1hander

Reply:
I'll swap them